internal/dag/status_test.go

```diff
@@ -2517,7 +2517,7 @@ func TestDAGStatus(t *testing.T) {
 			}: {
 				Object:      invalidResponseTimeout,
 				Status:      "invalid",
-				Description: "route.timeoutPolicy failed to parse: error parsing response timeout: unable to parse timeout string \"invalid-val\": time: invalid duration invalid-val",
+				Description: "route.timeoutPolicy failed to parse: error parsing response timeout: unable to parse timeout string \"invalid-val\": time: invalid duration \"invalid-val\"",
```

Comment:
Yep, I ran into this when I wrote these tests. Hazards of testing on precise error messages, I guess.

Reply:
It changed between Go 1.14 and Go 1.15, sigh.